api/os/os.cpp

```diff
@@ -146,8 +146,20 @@ pair<int, int> spawnProcess(string command, const string &cwd) {
         nextVirtualPid = 0;
     }
 
+    TinyProcessLib::Process::environment_type processEnv;
+
+    if (!options.envs.empty()) {
+        for (const auto& [key, value] : options.envs) {
+            #if defined(_WIN32)
+            processEnv[helpers::str2wstr(key)] = helpers::str2wstr(value);
```

Comment:
```suggestion
            processEnv[CONVSTR(key)] = helpers::str2wstr(CONVSTR(value));
```
You can utilize these macros to cast strings to appropriate format as per the operating system, as these macros are defined here.

https://github.com/neutralinojs/neutralinojs/blob/4f8c45e3640f8b9ae39b51a862899689a34256e3/helpers.h#L11